### ISSUE TITLE
[rush] "rush build --verbose" prints noisy diagnostic output such as "Attempting to load via rig"

### ISSUE THREAD

**Issue by octogonz** [microsoft/rushstack#2790]
## Summary

Starting around Rush 5.36.0, some irrelevant verbose logging is now displayed alongside the toolchain logs when invoking `rush build --verbose`.  Although this was introduced for the experimental build cache, the logging is printed even if the cache experiment is not enabled.

## Repro steps

1. Clone https://github.com/microsoft/rush-example/commit/a09a47d00c580e5ec11b6478f6e86a3607e1e75c
2. Run `rush install`
3. Run `rush build --verbose`

  **Expected result:** According to the docs, the `--verbose` flag will _"Display the logs during the build, rather than just displaying the build status summary."_  This is referring to the log output of the build shell scripts, NOT the log output of Rush itself. 

  **Actual result:**  Rush prints a bunch `heft-config-file` diagnostic logging that is not coming from the build shell scripts:

```
==[ my-toolchain ]=================================================[ 1 of 3 ]==

Config file "C:\Git\rush-example\apps\my-app\config\rush-project.json" does not exist. Attempting to load via rig.

No rig found for "C:\Git\rush-example\apps\my-app"

Config file "C:\Git\rush-example\libraries\my-controls\config\rush-project.json" does not exist. Attempting to load via rig.

No rig found for "C:\Git\rush-example\libraries\my-controls"

Config file "C:\Git\rush-example\tools\my-toolchain\config\rush-project.json" does not exist. Attempting to load via rig.

No rig found for "C:\Git\rush-example\tools\my-toolchain"

my-toolchain was skipped.

==[ my-controls ]==================================================[ 2 of 3 ]==
```

This is particularly annoying because:
1. The build cache experiment is disabled
2. The `my-toolchain` project was not even built -- it was skipped by the incremental analyzer.
2. Every newline is doubled

## Details

The `--debug` command line parameter normally is used to print internal diagnostics for Rush itself.  If these logs are important, perhaps they should be mapped to that parameter instead.

## Standard questions

Please answer these questions to help us investigate your issue more quickly:

| Question | Answer |
| -------- | -------- |
| `@microsoft/rush` globally installed version? |  5.47.0 |
| `rushVersion` from rush.json? |  5.47.0 |
| `useWorkspaces` from rush.json? | false |
| Operating system? |Windows  |
| Node.js version (`node -v`)? | 12.20.1 |

**Comment by atsikov** [comment]
I checked a few versions, and it looks like problem appeared exactly in `5.47.0`
`5.46.1` doesn't have extra messages in the output

**Comment by F3n67u** [comment]
> The --debug command line parameter normally is used to print internal diagnostics for Rush itself. If these logs are important, perhaps they should be mapped to that parameter instead.

@octogonz Rush seems don't have a `--debug` command line parameter. We should remove those `writeVerboseLine` when configuration file cannot find?

**Comment by octogonz** [comment]
> Rush seems don't have a `--debug` command line parameter.

It is a global parameter; you can see it by running `rush --help` (without any command).  And it must appear before the action name, e.g. `rush --debug build`.

**Comment by iclanton** [comment]
This should be fixed by #2839.
